### DISCUSSION TITLE
how can i assign custom location for lora and models ?

### DISCUSSION THREAD

**Discussion by Trial7777777** [lllyasviel/Fooocus#132] [General]
how can i assign custom folder location for lora and models ?

**Comment by ajpixs** [comment]
I would love to see this as a option for loras, models, embedding & hypernets , if it doesn't exist. Last thing I need is ANOTHER folder filled 500gbs of files.

**Comment by KeulzZz** [comment]
You can use symbolic link to have only one folder containing models for several softwares.
It should look like something like that in powershell :
New-Item -ItemType Junction -Path "path-to-model-folder-in-fooocus" -Target "path-to-folder-with-all-your-models"

**Comment by mirek190** [comment]
or under CMD or PS

mklink "C:\Path\Symlink_File.txt" "F:\Real_File.txt"

**Comment by tjohn88** [comment]
Try to change this one (in the .\Fooocus\modules\path.py): 

modelfile_path = os.path.abspath(os.path.join(os.path.dirname(__file__), '../models/checkpoints/'))
lorafile_path = os.path.abspath(os.path.join(os.path.dirname(__file__), '../models/loras/'))
temp_outputs_path = os.path.abspath(os.path.join(os.path.dirname(__file__), **'E:/!mySD-Projects/!Outputs/'**))

it works for me (I've changed only output path, but you can try change models path too).
